Userland/Libraries/LibGfx/ImageFormats/JPEGXLLoader.cpp

```diff
@@ -2743,6 +2743,13 @@ static ErrorOr<Frame> read_frame(LittleEndianInputBitStream& stream,
         frame.height = ceil(static_cast<double>(frame.height) / frame.frame_header.upsampling);
     }
 
+    // "If lf_level > 0 (which is also a field in frame_header), then
+    // width = ceil(width / (1 << (3 * lf_level))) and height = ceil(height / (1 << (3 * lf_level)))."
+    if (frame.frame_header.lf_level > 0) {
+        frame.width = ceil(static_cast<f32>(frame.width) / (1 << (3 * frame.frame_header.lf_level)));
```

Comment:
ceil_div() would obviate the need to cast to f32 and be otherwise equivalent, right?